app/javascript/packs/articlePage.jsx

```diff
@@ -1,67 +1,70 @@
-import { h, render } from 'preact';
-import { Snackbar, addSnackbarItem } from '../Snackbar';
+import { h, render } from 'preact'; 
```

Comment:
tiny detail: there are a lot of whitespace only changes in this file. It might be your settings in the editor.

It added end of line space and remove the end of line space on the last line.

Not sure what editor you use but in Sublime Text 3 there's a setting called `trim_trailing_white_space_on_save` which will solve this automatically on save

Reply:
We need to put that embed on submit. I'll trim those ugly whitespaces :(

Reply:
I'm searching the appropriate event.

Reply:
@rhymes I made it work for both comment submit and preview. Do you know which file handles the edit part? As far as I know, it's the only remaining case : edit comment and then preview (url: /comments/XXX/edit). I managed the preview when on page article.

If you have any hint, I can fix that when I get the time. Thanks.

Reply:
Awesome work @jmau111! 

You can find the comment preview code here: 

https://github.com/forem/forem/blob/92d8bfe31a5a7b89dddf589564a059026987b0d6/app/assets/javascripts/initializers/initializeCommentPreview.js#L67

Reply:
@rhymes should work for that edit comment preview too, I've noticed many places where gists should be embedded, such as when the user clicks on "dismiss". The latest commit handles that.

Hope I'm not missing any case, for now it covers: 

* single post
* preview post
* single comment
* preview comment
* dismiss edit comment

I had to add a custom pack `commentsDisplay.jsx` to load helper for comments.